.gitignore

```diff
@@ -1,5 +1,28 @@
+# raw images are too large for Github so it is ignored
 0.download-data/Images
-1.preprocessing-data/IC_Corrected_Images
+
+# ignore pycache 
 __pycache__
+
+# test files that are not relevant to the pipelines
 1.preprocessing-data/test.ipynb
 example.csv
+
+# old raw images that were being used for analysis but is not used 
+0.download-data/Old_Images
+
+# old IC images from previous data that is not used
+1.preprocessing-data/Old_IC_Images
+
+# IC corrected that totalling 9.5 GB -> too large for GitHub
+1.preprocessing-data/IC_Corrected_Images
```

Comment:
Could these lines be condensed into an overarching ignore pattern such as `*.tif` or `*.png`?

Reply:
Oh, thats a good idea! Since none of the images are put on Github anyway!